helper.sh

```diff
@@ -35,6 +35,7 @@ init_boot ()
     touch "${DIR}"/secrets/acme.json # to avoid making it think it's a folder
     chmod 600 "${DIR}"/secrets/acme.json # Traefik doesn't let ACME challenge go through otherwise
 
+    touch "${DIR}"/oms-global/docker/traefik/traefik.toml.template # to avoid making it think it's a folder
```

Comment:
I meant to touch the file not yet existing, i.e. `"${DIR}"/oms-global/docker/traefik/traefik.toml`

Reply:
that's where the misunderstanding came from :)

Reply:
Whoops, my fault. I intended to touch the file and not the template, but just copied the wrong string lol.
Will fix today.